rgw/admin/README.md

```diff
@@ -24,9 +24,6 @@ func main() {
         panic(err)
     }
 
-    // To enable debug requests
-    // co.Debug = true
```

Comment:
For my understanding, here end-user need to define the debug API interface, right?. Do we need to mention that here?

Reply:
Yes, the consumer needs to define a new type for mutating the HTTP client. I'm not sure if it's worth mentioning...

Reply:
If I was going to mention debugging - I'd just note that the http client type is an interface such that you can mock it or wrap it to fit your needs. And perhaps point to the test code with a hyperlink for an example.

Reply:
Thanks @leseb for clarification. I am okay with or without doc changes